docs/installation/local.md

```diff
@@ -2,169 +2,177 @@
 
 ## Dependencies required to run Orga Server
 
-* Python 3.7
-* Postgres
-* OpenSSL
+-   Python 3.7
+-   Postgres
+-   OpenSSL
 
-### For mac users
-```sh
-brew install postgresql
-````
-
-### For debian-based linux users
-```sh
-sudo apt-get update
-sudo apt-get install postgresql postgresql-contrib libssl-dev
-```
-
-## Steps
+## Get a copy of source code
 
-Make sure you have the dependencies mentioned above installed before proceeding further.
+- Clone the Open Event Server repository (from the development branch) and `cd ` into the directory.
 
-* **Step 0** - Clone the Open Event Server repository (from the development branch) and ```cd ``` into the directory.
 ```sh
-git clone -b development https://github.com/fossasia/open-event-server.git
+git clone https://github.com/fossasia/open-event-server.git
 cd open-event-server
 ```
-**Note :** If you want to contribute, first fork the original repository and clone the forked repository into your local machine followed by ```cd``` into the directory
+
+- If you want to contribute, first fork the original repository to your GitHub profile and clone that fork into your local machine, followed by `cd` into the directory
+
 ```sh
-git clone https://github.com/USERNAME/open-event-server.git
+git clone git@github.com:USERNAME/open-event-server.git
```

Comment:
It may not be necessary that beginners have used SSH method for GitHub authentication. Most beginners use HTTPS

Reply:
We should teach beginner the good practice. I already mention the benefit of SSH over HTTPS in README (no need to type password again and again).

Reply:
@hongquan It would be one more barrier to cross for a beginner before starting to contribute here. Also, I think it's fine if they use `https` at the very beginning.

Reply:
We can provide an additional section below saying if you are using SSH, use the command below.
Please realize the level of beginners approaching the repo is such that even when the README explicitly states that Python 3.7 is required to run the project, they run it on 3.6 or 3.8. Or in frontend repo, they miss a command before starting a server and regularly ask what's the issue. If we write a command in README which will not work right away for most users, they'll most likely not set up SSH with GitHub, they'll either complain or quit. Pragmatism beats perfectionism here, unfortunately.